Cargo.toml

```diff
@@ -88,6 +88,7 @@ members = [
     "sdk/program",
     "send-transaction-service",
     "stake-accounts",
+    "stake-program-test",
```

Comment:
nit: let's put this in `programs/stake-tests` for consistency with address lookup table tests